docs/pages/User/Configure/configure.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Configuration
+weight: 1
+---
+
+### Edit your cluster name
+
+The ClusterName is your cluster's nickname, a simple and understandable name that the other clusters can see when they 
```

Comment:
@palexster Would it be possible to avoid "hard" line termination in documentation? It would make the review much more difficult. I would suggest to write text with *one sentence per line* in order to facilitate the review.

Reply:
```suggestion
The ClusterName is your cluster's nickname, a human-friendly string that identifies your cluster when discovering or peering with other parties. 
```

Reply:
Hi @frisso! I am not the author nor I have approved the contents.

Reply:
yes sure